### DISCUSSION TITLE
[Script request] Flame Dashboard

### DISCUSSION THREAD

**Discussion by Deepshah0308** [community-scripts/ProxmoxVE#165] [Request script] [answered]
### Application Name

flame

### Website

https://github.com/pawelmalak/flame

### Description

Flame is self-hosted startpage for your server. Its design is inspired (heavily) by [SUI](https://github.com/jeroenpardon/sui). Flame is very easy to setup and use. With built-in editors, it allows you to setup your very own application hub in no time - no file editing necessary.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Mellowlynx** [comment, marked as answer]
Duplicate of #115 
Make sure you really search before just posting!
